modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/regions/NL.json

```diff
@@ -1,10 +1,14 @@
 [
+	{
+		"name": "Aruba",
+		"regionCode": "AW"
+	},
 	{
```

Comment:
@wanderlast Shouldn't BQ1 be moved under the BQ.json file? Please see https://www.iso.org/obp/ui/#iso:code:3166:BQ. Thanks FYI @riccardo-alberti